terraform/stacks/external/stack.tf

```diff
@@ -13,3 +13,19 @@ module "limit_check_user" {
   source = "../../modules/iam_user/limit_check_user"
   username = "limit-check-${var.stack_description}"
 }
+
+module "operators" {
+  source = "../../modules/operators"
+}
+
+resource "aws_iam_policy_attachment" "admin" {
+  name = "admin"
+  count = "${var.stack_description == "production" ? 1 : 0}"
+  policy_arn = "arn:aws:iam::aws:policy/AdministratorAccess"
+  roles = [
+    "${concat(
+      module.operators.operators,
+      list("terraform-provision")
```

Comment:
I have no idea if cross-partition account delegation works, but should we give that a go rather than keeping this AWS cred around? Or we can punt it for now and make a card to try it out in a future sprint...